src/core/models/dto/delegatesDTO.ts

```diff
@@ -1,6 +1,6 @@
 import type { ActivityFeedDto, BudgetStatementDto } from './coreUnitDTO';
 
-export interface DelegatesDto {
+export interface DelegatesReportDto {
```

Comment:
This should keep its name `DelegatesDto` as it is a DTO for the delegates in general, no matter where it is used (like the CoreUnitsDto)

Reply:
done